### ISSUE TITLE
PIQA_PREDECESSOR_IDS doesn't do anything

### ISSUE THREAD

**Issue by compubasejovl** [MTAGRatingen/Quality-Assurance-Apex-Plug-In#1]
The value entered into PIQA_PREDECESSOR_IDS have no effect. 

Example from predefined rules, when item has condition "Never" the error from rule ID 1 is displayed, when the item also has no help text the error is also displayed. 

The help text error should not be displayed because the ID from condition "Never" is entered into PIQA_PREDECESSOR_IDS

Example application:
https://apex.oracle.com/pls/apex/f?p=87554

**Comment by OliverLemm** [comment]
Yes thats correct. I got problems while using the predecssor_ids and therefor the last version was without using the attribute. After october i will have a look into enhancements.

Thanks for your comment
